src/WebDriverBiDi/Session/SubscribeCommandParameters.cs

```diff
@@ -23,6 +23,28 @@ public SubscribeCommandParameters()
     {
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="SubscribeCommandParameters"/> class.
+    /// </summary>
+    /// <param name="events">The list of events to which to subscribe or unsubscribe.</param>
+    /// <param name="contexts">The list of browsing context IDs for which to subscribe to or unsubscribe from the specified events.</param>
+    public SubscribeCommandParameters(string[] events, string[] contexts)
```

Comment:
Would one constructor using `IList<string>` args work better? That would cover both `List<string>` and `string[]`, I think (I may be wrong).

Reply:
I think that can work. 👍

Reply:
Done!